lib/dialects/sqlite/query.js

```diff
@@ -214,8 +214,9 @@ Query.prototype.formatError = function (err) {
         , message = 'Validation error';
 
       fields.forEach(function(field) {
+        var fieldMessage = self.getUniqueConstraintErrorMessage(field);
```

Comment:
No need for the intermediate variable is there? Can just put the call directly in the code blow.